hooks/pre_build_centos

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -ex
```

Comment:
What is this file for?

Reply:
It is used for CentOS, but this is future.